packages/dm-core-plugins/src/form/Form.tsx

```diff
@@ -43,10 +48,12 @@ export const Form = (props: TFormProps) => {
       <FormProvider {...methods}>
         <RegistryProvider onOpen={onOpen} idReference={idReference}>
           <form onSubmit={handleSubmit}>
-            <Stack spacing={2}>
-              {type && (
-                <ObjectField config={config} namePath={namePath} type={type} />
-              )}
+            <Stack spacing={2} style={{ padding: '1rem 0' }}>
+              <AttributeList
+                namePath={namePath}
+                config={config}
+                blueprint={blueprint}
+              />
```

Comment:
I have not tested it, but I think with this change we will lose the functionality to select a custom widget in the RecipeLink.
 For example, whether to display a "textWidget" (single line text input) or a "TextareaWidget" (multi line text input)

see for example employee.recipe.json

![image](https://github.com/equinor/dm-core-packages/assets/69512295/36cc3c9e-866b-4fe2-a733-60caa6021620)